cfme/tests/automate/test_vmware_methods.py

```diff
@@ -105,7 +116,7 @@ def _get_disk_count():
             properties=("Datastore Allocation Summary", "Number of Disks")).strip())
     original_disk_count = _get_disk_count()
     toolbar.select(testing_group.text, button.text)
-    fill("input#size", "1")
+    fill(Input("size"), "1")
```

Comment:
Any chance of using a real locator here, instead of something buried in a test? :)